pkg/services/ngalert/notifier/file_store.go

```diff
@@ -8,6 +8,7 @@ import (
 	"path/filepath"
 
 	alertingNotify "github.com/grafana/alerting/notify"
+	"github.com/prometheus/alertmanager/cluster/clusterpb"
```

Comment:
One last thing: we've gone at length to avoid direct Alertmanager imports here in Grafana and make them all come through grafana/alerting - I don't think this type is aliased to let's make sure we create a task to do so eventually.